classifier/CompoundStreetClassifier.js

```diff
@@ -15,6 +15,12 @@ class CompoundStreetClassifier extends WordClassifier {
       // this removes suffixes such as 'r.' which can be ambiguous
       minlength: 3
     })
+
+    libpostal.load(this.suffixes, ['de', 'nl'], 'concatenated_suffixes_inseparable.txt', {
```

Comment:
question: is `de` required here to get `nl` working or is this just nice to have since it solves the same problem in another language?

Reply:
It's the latter: it solve the same problem ;-)